scrapper.py

```diff
@@ -1,35 +1,81 @@
+import argparse
+import re
 import requests
 from bs4 import BeautifulSoup
-from sys import argv
-import re
 
-def terminate_script(exit_code: int):
-    OUTPUT_MESSAGES = {1:"You missed an argument. It's expected a URL that should be grabbed.",
-        2:"Argument have to begin from 'https://krisha.kz'.",
-        3:"Too many arguments. It's expected just one argument which is URL to be grabbed.",
-        4:"Page is not available. Check connection and restart the script."}
+def cli_parser() -> argparse.ArgumentParser:
+    """ Function parse a command line. 
+        Function format: python scrapper.py [-h] [url] [-ff file_with_url], where:
+            - [-h] or [--help] optional argument. Description of ArgumentParser will be printed;
+            - [url] positional argument site utl for scrapping;
+            - [-ff file_with_url] or [--fromfile file_with_url] optional argument. PATH to file with site url.
+        Function return ArgumentParser object. 
+    """
+
+    cli_parser = argparse.ArgumentParser(description='Scrapper from krisha.kz')
+    cli_parser.add_argument('url', default=None, nargs='?',
```

Comment:
@ProvVlad , please get acquainted with this argparse capability https://docs.python.org/3/library/argparse.html#mutual-exclusion. I suggest to change parser initialization in the following way:
`group = cli_parser.add_mutually_exclusive_group()
    group.add_argument('url', default=None, nargs='?',
                            help='site url for scrapping. Have to start with symbols "https://krisha.kz"'
                            )
    group.add_argument('-ff', '--fromfile',  dest='file_with_url',
                            help='''file with site url for scrapping.
                            You have to specify the full path to file 
                            or file have to be on the same folder'''
                            )
    args = cli_parser.parse_args()`